index.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>mint login form clone</title>
+</head>
+<body>
+    <main>
+        <div id="sign-in-btn-info">
+            <div id="text-info">
+                <div id="question">
+                    <p>Already have an Intuit account?</p>
+                </div>
+                <div id="user-id-password">
+                    <a href="#">
+                        <p> I forgot my User ID or Password</p>
+                    </a>
+                </div>
+            </div>
+            <button  id="sign-in-btn">
+                <i class="fa fa-lock"></i>
+                <span>Sign in</span>
+            </button>
+        </div>
+    <div class="header">
+        <div class="header-links">
+            <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D08497889761544046421391270368961118431%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1562778150%26ivid%3D538f0975-5631-4a09-93f2-0c8809828c71"><img src="img/intuit.png" alt="intuit"></a>
+
+            <div id="link-imgs">
+                <a href="https://www.mint.com/"><img src="img/mint.png" alt="mint"><span class="product-name">mint</span></a>
+                <a href="quickbooks.intuit.com"><img src="img/quickbooks.png" alt="quickbooks"><span class="product-name">quickbooks</span></a>
+                <a href="turbotax.intuit.com"><img src="img/turbotax.png" alt="turbotax"><span class="product-name">turbotax</span></a>
+
+            </div>
+        </div>
+    </div>
+    <div id="login-container">
+        <div class="form-header">
+            <h1>Create an Intuit account</h1>
+            <p>One account for everything Intuit, including Mint.<br><a href="#">Learn more</a></p>
+            <p id="alt-login-question">Already have an account? <a href="#">Sign In</a></p>
+        </div>
+        <form id="login-form" action="#" method="post">
+
+            <label for="email-ctl">
```

Comment:
`AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)